frontend/widgets/OBSBasicControls.cpp

```diff
@@ -121,10 +121,19 @@ void OBSBasicControls::StreamingStarting(bool broadcastAutoStart)
 	}
 }
 
+namespace {
+void setButtonActiveState(QPushButton *button, bool active)
```

Comment:
We have an existing helper function in qt-wrappers.cpp called `setClasses` for setting the `class` dynamic property

Reply:
Updated to use `setClasses`.